src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+public static class MicrobeInternalCalculations
+{
+    public static float CalculateSpeed(IEnumerable<OrganelleTemplate> organelles, MembraneType membraneType,
+        float membraneRigidity)
+    {
+        float microbeMass = Constants.MICROBE_BASE_MASS;
+
+        float organelleMovementForce = 0;
+
+        foreach (var organelle in organelles)
+        {
+            microbeMass += organelle.Definition.Mass;
+
+            if (organelle.Definition.HasComponentFactory<MovementComponentFactory>())
+            {
+                if (organelle.Orientation == 3)
+                {
+                    organelleMovementForce += Constants.FLAGELLA_BASE_FORCE
+                        * organelle.Definition.Components.Movement.Momentum / 100.0f;
+                }
+            }
+        }
+
+        float baseMovementForce = Constants.CELL_BASE_THRUST *
+            (membraneType.MovementFactor - membraneRigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER);
+
+        float finalSpeed = (baseMovementForce + organelleMovementForce) / microbeMass;
```

Comment:
I guess the approximation is fine, but I truly find it disturbing that a speed is computed as a force over a mass (which should be a speed/s, that is an acceleration).

Reply:
"fine" is good enough for me right now :)

(this is also code from before the PR)

Reply:
I'm still pretty wary of this legacy code now. But I think there would be much to change, so we should perhaps leave it aside for now.